README.md

```diff
@@ -31,5 +31,5 @@ The [project homepage](http://janusgraph.org) contains more information on Janus
 
 ## Contributing
 
-Please see [`CONTRIBUTING.md`](CONTRIBUTING.md) for more information, including
```

Comment:
The CONTRIBUTING.md file is the one shown to the user when submitting a PR, I renamed the existing one to GETTING_INVOLVED.md and added a new CONTRIBUTING.md (part of this PR). Maybe have a look at both versions and see if you think it makes sense as well.